lib/entry-index.js

```diff
@@ -335,6 +270,8 @@ function _bucketEntries (data, filter) {
       // Entry is corrupted!
       return
     }
+    // coverage disabled here, no need to test with an entry that parses to something falsey
```

Comment:
How was this passing before?

Reply:
i have no idea. i stopped short of bisecting every single test i deleted trying to find it before i gave up on it